deployment/docker/rules/alerts-health.yml

```diff
@@ -128,3 +128,21 @@ groups:
           summary: "Some rows are rejected on \"{{ $labels.instance }}\" on ingestion attempt"
           description: "Ingested rows on instance \"{{ $labels.instance }}\" are rejected due to the
             following reason: \"{{ $labels.reason }}\""
+
+      - alert: TooHighQueryLoad
+        expr: increase(vm_concurrent_select_limit_timeout_total[5m]) > 0
```

Comment:
```suggestion
        expr: increase(vm_concurrent_select_limit_timeout_total[1m]) > 0
```
should we reduce the lookbehind window to a smaller value like `1m`?
with 5m, if `vm_concurrent_select_limit_timeout` is reached once every 5m over a 15m period, the alert is triggered but that doesn't seem particularly concerning.

Reply:
`5m` is default resolution across all other alerts. It is also safe for Prometheus, as its increase is vulnerable to amount of datapoints that fall into [look-behind-window]. `[1m]` interval for Prometheus would require scrape_interval to be at least 15s.